bigchaindb/commands/bigchaindb.py

```diff
@@ -313,63 +237,12 @@ def create_parser():
     start_parser = subparsers.add_parser('start',
                                          help='Start BigchainDB')
 
-    start_parser.add_argument('--dev-allow-temp-keypair',
```

Comment:
`--dev-allow-temp-keypair` and `--dev-start-rethinkdb` were undocumented and quasi-secret, nevertheless they existed as valid CLI arguments. I wonder if you should mention removing them in the proposal, i.e. in PR #1976

Reply:
I added a bullet point about removing these in the comment description of this PR (at the top of this page).